src/GenericSpectator/Assets/Scripts/PlatformDeviceBootstrapper.cs

```diff
@@ -0,0 +1,49 @@
+﻿using UnityEngine;
+
+public class PlatformDeviceBootstrapper : MonoBehaviour
```

Comment:
Missing namespace.

Reply:
Thanks!

---
In reply to: [337246998](https://github.com/microsoft/MixedReality-SpectatorView/pull/250#discussion_r337246998) [](ancestors = 337246998)